src/command/client/search/interactive.rs

```diff
@@ -327,13 +342,28 @@ impl State {
         let input = Paragraph::new(input);
         f.render_widget(input, chunks[2]);
 
+        let selected = self.results_state.selected();
+        let command = if results.is_empty() {
+            ""
+        } else {
+            &results[selected].command
+        }
+        .chars()
+        .collect::<Vec<char>>()
+        .chunks(preview_width.into())
+        .map(|chunk| chunk.iter().collect::<String>())
+        .collect::<Vec<String>>()
+        .join("\n");
```

Comment:
this is me being grumpy about extra allocations 😅
```suggestion
        let command = if results.is_empty() {
            String::new()
        } else {
            use itertools::Itertools as _;
            let s = &results[selected].command;
            s.char_indices()
                .step_by(preview_width.into())
                .map(|(i, _)| i)
                .chain(Some(s.len()))
                .tuple_windows()
                .map(|(a, b)| &s[a..b])
                .join("\n")
        };
```

Reply:
Although the original is a bit easier to understand ;)

Reply:
I've worked hard to make the renderer efficient!